File: app/src/main/java/com/example/gitbusca/model/Owner.java
package com.example.gitbusca.model;

public class Owner {
    private String login;
    private String avatar_url;

    public Owner(String login, String avatar_url) {
        this.login = login;
        this.avatar_url = avatar_url;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public String getAvatar_url() {
        return avatar_url;
    }

    public void setAvatar_url(String avatar_url) {
        this.avatar_url = avatar_url;
    }
}

File: app/src/main/java/br/com/repository/model/Owner.kt
package br.com.repository.model

import android.databinding.BaseObservable
import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Owner(@SerializedName("login")
            var name: String,
            @SerializedName("avatar_url")
            var urlImage: String) : BaseObservable(), Parcelable {


}